Sources/Logto/Types/IdTokenClaims.swift

```diff
@@ -7,17 +7,20 @@
 
 import Foundation
 
-public struct IdTokenClaims: Codable, Equatable {
+public struct IdTokenClaims: UserInfoProtocol {
     public let sub: String
     public let atHash: String?
     public let aud: String
     public let exp: Int64
     public let iat: Int64
     public let iss: String
 
-    // Scope `profile`
+    // Protocol
     public let name: String?
+    public let picture: String?
     public let username: String?
-    public let avatar: String?
-    public let roleNames: [String]?
```

Comment:
We still have roleNames in idToken...

Reply:
it is useless now and shouldn't exist once RBAC implemented

Reply:
OK, then we need to make this change in other SDKs as well

Reply:
no need for now. when we implement RBAC we'll find the disharmony in SDK and remove it